pkg/kubelet/network/cni/cni.go

```diff
@@ -170,6 +189,24 @@ func (network *cniNetwork) addToNetwork(podName string, podNamespace string, pod
 	return res, nil
 }
 
+func (plugin *cniNetworkPlugin) getDefaultNetwork() *cniNetwork {
+	plugin.RLock()
+	defer plugin.RUnlock()
+	return plugin.defaultNetwork
+}
+
+func (plugin *cniNetworkPlugin) setDefaultNetwork(n *cniNetwork) {
+	plugin.Lock()
+	defer plugin.Unlock()
+	plugin.defaultNetwork = n
+}
+
+func (plugin *cniNetworkPlugin) Status() error {
+	if plugin.getDefaultNetwork() == nil {
+		return errors.New("cni config unintialized")
+	}
+	return nil
+}
```

Comment:
Nit: these (*cniNetworkPlugin) methods are intermixed with (*cniNetwork) methods. Maybe move them inline with rest of calls.